src/app/utils/chart-utils.ts

```diff
@@ -4,35 +4,42 @@ import isSameMonth from 'date-fns/isSameMonth'
 import startOfMonth from 'date-fns/startOfMonth'
 import { GetLayerStatsTxVolumeParams, type TxVolume, type ActiveAccounts } from '../../oasis-nexus/api'
 
+const fiveMinutesWindowSize = 60 * 5
+const oneDayWindowSize = 24 * 60 * 60
+
 export enum ChartDuration {
   TODAY = 'TODAY',
   WEEK = 'WEEK',
   MONTH = 'MONTH',
   ALL_TIME = 'ALL_TIME',
 }
 
-export const dailyLimitWithoutBuffer = (60 / 5) * 24 // full day for 5 minutes buckets,
+export const dailyLimitWithoutBuffer = (60 / 5) * 24 // full day for 5 minutes windows
 
 export const durationToQueryParams = {
   [ChartDuration.TODAY]: {
-    bucket_size_seconds: 60 * 5,
+    window_size_seconds: fiveMinutesWindowSize,
+    window_step_seconds: fiveMinutesWindowSize,
```

Comment:
Can you add comment about the difference

Reply:
done

Reply:
Oh oops, hovering on them already explains the difference. It shows field docs because of `satisfies { [duration in ChartDuration]: GetLayerStatsTxVolumeParams }`